### DISCUSSION TITLE
About chart Y-axis

### DISCUSSION THREAD

**Discussion by Rainproof** [frangoteam/FUXA#347] [Ideas]
It would be nice to color the numerical values of the Y-axes in the colors of the corresponding charts if the axes > 1.
![03-05-2022 1-48-15](https://user-images.githubusercontent.com/19856627/166339679-34cffaf6-cb68-468e-963d-138bba45fddf.jpg)
![03-05-2022 1-49-38](https://user-images.githubusercontent.com/19856627/166339691-10b4a916-c84b-4273-9307-2af4f4e7f483.jpg)

**Comment by unocelli** [comment]
I find it cool, but how do we manage if there are two lines with different color on the same axis.

**Reply by Rainproof** [reply]
I thought maybe so
![03-05-2022 1-49-38-1](https://user-images.githubusercontent.com/19856627/167013994-8e155b85-e5f9-4b85-bf04-18ead4446d44.jpg)

**Reply by unocelli** [reply]
:confused: I this case it is out of my hands, I'm using [uplot](https://github.com/leeoniya/uPlot) and don't think it is possible.

**Reply by leeoniya** [reply]
you can definitely set axis colors

https://leeoniya.github.io/uPlot/demos/axis-control.html

**Reply by unocelli** [reply]
OK, I will add the possibility to settings the colour of the Y-axes, so that you can decide for yourself which colour.

**Reply by Rainproof** [reply]
The issue with the axis colors came up when I tried the possibility of adding multiple plots with different ranges of values. If the two axes are with colors that match the colors of the graphs, then it is easier to visually navigate. I assumed that the program itself would assign the color of the axis according to the color of its graph.

**Reply by Rainproof** [reply]
But the colors of the chart axes are very secondary  ) . It is logical to place them on charts with similar ranges of values. Much more interesting now is the connection with external databases  )
